TransferMethod/Sources/ListTransferMethodController.swift

```diff
@@ -97,8 +97,8 @@ final class ListTransferMethodController: UITableViewController {
 
     override public func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
         if !(presenter.sectionData[indexPath.row].isPrepaidCard()) {
-            showConfirmationAlert(title: "mobileRemoveEAconfirm".localized(),
-                                  message: "mobileAreYouSure".localized(),
+            showConfirmationAlert(title: "mobileAreYouSure".localized(),
+                                  message: "",
```

Comment:
Can you attach a screenshot in Jira ticket, for this change? Thank You!

Reply:
Attached screenshot in Jira.